modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -68,106 +94,153 @@ mirror:
             minVersion: 3.68.0
 ----
```

Comment:
this `ImageSetConfiguration` file is not working , when mirror the catalog, we must include the default channel , for example `rhacs-operator` the default channel is table , so the imagesetconfigure should like : 
apiVersion: mirror.openshift.io/v1alpha2
kind: ImageSetConfiguration
storageConfig:
  local:
    path: /home/user/metadata
mirror:
  operators:
    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.13
      packages:
        - name: rhacs-operator
          channels:
            - name: stable
            - name: latest
              minVersion: 3.74.1

Reply:
I saw we have example for "Use case: Including the default Operator channel" , I think we could remove this part , this example is incorrect.

Reply:
So for now I have just updated to add "stable" to the list here.

Will let @sherine-k and @aguidirh weigh in on whether we should remove this example completely instead (in favor of the new default operator channel example).

Reply:
How about keeping it simple, and using channel stable in this example. Available versions in stable version are:
```
 ./bin/oc-mirror list operators --catalog=registry.redhat.io/redhat/redhat-operator-index:v4.13 --package=rhacs-operato
r --channel=stable
VERSIONS
4.0.0
4.0.1
4.0.2
```